src/containers/dummy-leaderboard/DummyLeaderboard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import LeaderboardTabs, {
+  TabInfo,
+} from '../../components/leaderboard/leaderboard-tabs/LeaderboardTabs';
+import { LeaderboardItem } from '../../components/leaderboard/leaderboard-tab/LeaderboardTab';
+
+const dummyTabs: TabInfo[] = [
+  {
+    name: 'one',
+    content: [
+      {
+        rank: 1,
+        name: 'Jack',
+        rightSide: 100,
+        collapseContent: 'hello',
+      },
+      {
+        name: 'Floris',
+        rightSide: 90,
+      },
+      {
+        name: 'Breanna',
+        rightSide: 80,
+      },
+      {
+        name: 'Aaron',
+        rightSide: 80,
+      },
+      {
+        name: 'Sophie',
+        rightSide: 80,
+      },
+      {
+        name: 'Owen',
+        rightSide: 80,
+      },
+      {
+        name: 'Will',
+        rightSide: 7,
+      },
+    ],
+  },
+];
+
+const DummyLeaderboard: React.FC = () => {
```

Comment:
Let's rename this to `VolunteerLeaderboard` so we don't have to change it later.